playground/Playground.js

```diff
@@ -15,7 +17,8 @@ export function Playground() {
       <div>
         {/* Add the code you want to test here */}
         <Button href="https://google.com">APPLY</Button>
-        <Link variant="success" href="https://google.com">New link</Link>
+        <Link variant="primary" href="https://google.com">New link</Link>
```

Comment:
maybe dont commit the playground? I shouldnt of committed before too.. I'll remove the other stuff later